certified-connectors/ProcessStreet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,931 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateWorkflowRunRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/CreateWorkflowRunResponse"
+            }
+          },
+          "400": {
+            "description": "Invalid value for: body"
+          }
+        }
+      }
+    },
+    "/workflow-runs/{workflowRunId}/form-fields": {
+      "get": {
+        "summary": "List all form field values",
+        "x-ms-summary": "List all form field values",
+        "description": "Allows to get a batch of form field values for a workflow run.\nForm field values are paged, meaning the links section in the\nresponse must be used to get the next/previous batch of form field values.",
+        "operationId": "listFormFieldValues",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "workflowRunId",
+            "x-ms-summary": "Workflow Run Id",
+            "x-ms-url-encoding": "single",
+            "in": "path",
+            "description": "The ID of the Workflow Run",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "_",
+            "description": "The page number to fetch fields for.",
+            "x-ms-summary": "Page Number",
+            "x-ms-url-encoding": "single",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ListFormFieldValuesResponse"
+            }
+          },
+          "400": {
+            "description": "Invalid value for: query parameter _"
+          }
+        }
+      }
+    },
+    "/workflow-runs/{workflowRunId}": {
+      "put": {
+        "summary": "Update a workflow run",
+        "x-ms-summary": "Update a workflow run",
+        "description": "Updates a workflow run.",
+        "operationId": "updateWorkflowRun",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "workflowRunId",
+            "x-ms-summary": "Workflow Run Id",
+            "x-ms-url-encoding": "single",
+            "in": "path",
+            "description": "The ID of the Workflow Run",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateWorkflowRunRequest"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
```

Comment:
This is an endpoint of our API that returns no content (as the 204 No content HTTP status code indicates). 

Can this be an exception to that case even though it's not dynamic?

Reply:
@g-piffa for the first time this can be an exception, but in the next update please provide the correct schema. Maybe you'll be able to update the API to return at least a success message. Please resolve the conversation after reading the message.

Reply:
Thanks!